docs/robots-txt.md

```diff
@@ -0,0 +1,16 @@
+# Robots.txt
+
+The SEO module will read a custom `robots.txt` file from `/config/robots.txt` in your project's root directory.
+
+The contents of that file will be appended to the `robots.txt` file generated by the CMS. Other components and plugins may add content to `robots.txt` via the `robots_txt` filter.
```

Comment:
I'd add an example of the `robots.txt` file, as a comparison / counter example to using the filter.

Also, I think it would be good to clarify that this content is accessed as `site_url/robots.txt`